api/services/call_log_service.py

```diff
@@ -98,7 +98,7 @@ def create_call_logs(self, jsonData):
                 else self.flow_category,
                 created_on=jsonData["log_created_on"]
                 if jsonData.get("log_created_on", None)
-                else datetime.now(),
+                else datetime.utcnow(),
```

Comment:
`datetime.now()` gives the system current time. In cloud function the system default timezone is UTC so it was working fine till now but we should use `utcnow()` instead

Reply:
Also the created_on and updated_on field should be in UTC.

Reply:
Yea, make sense to use UTC explicitly. Though, the default server date-time is in UTC only.

Also, as you are updating it here, just checking if you have checked at other places as well? It will be hard to maintain two different time zone on different machines.